TodoApp/test/TodoApp.Services.Tests/Comparers/ItemEqualityComparer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework.Constraints;
+using TodoApp.Contracts.Models;
+
+namespace TodoApp.Services.Tests.Comparers
+{
+    internal static class ItemEqualityComparerWrapper
+    {
+        private static Lazy<ItemEqualityComparer> LazyItemEqualityComparer => new Lazy<ItemEqualityComparer>();
+
+        private sealed class ItemEqualityComparer : IEqualityComparer<Item>
```

Comment:
If you mind R# complaining about the class being unused, either create a private constructor for it (it is usual approach to singletons) – that will force you to use the `ctor` in the lazy's initialization (and some other things, making the comparer "more classic" singleton) or omit the rule for this very line – either way will be a statement, that is always better than constantly "overlooking" R#'s warnings 😉 

Note: If you went for singleton, tell me, I will show one extra trick with `Lazy` for free 😃